### ISSUE TITLE
Feature Request: Easier view of how components fit together

### ISSUE THREAD

**Issue by davejlong** [hominoids/SBC_Case_Builder#15]
In developing accessory layouts it can be difficult to determine if there's enough spacing for things. For example, putting a fan into a case. It's hard to tell if there's actually enough room between the case and the board for a fan. The best way I've found to do it, is to render the Model view to STL, load it into SuperSlicer, slice the model and move through layers to see how things look.

Perhaps we could make it so that when looking at the model view, accessories like fans, speakers, etc. would render the actual shape. Then when I render the case in Model view, and look at the Wireframe view in OpenSCAD, it would be easy to see any overlap between the accessory and the SBC.

Perhaps we could even just use the [NopSCAD](https://github.com/nophead/NopSCADlib) library for generating the objects.

**Comment by hominoids** [comment]
If I understand what your requesting, to view accessory models in model view, it can be done already but there is not a fan model yet.  Take a look at the hc4 or h2 accessory entries for examples of model accessories as part of the model view.  

I have used [cutaway views](https://forum.odroid.com/viewtopic.php?f=53&t=43948&start=19) (see h2 button cutaway) for fit issues and there may be a way to add them to the VIEW tab for cases of limited pieces, supplementing the move functions.  So for instance, since the shell case only has a top and bottom to move, use a cutaway on demand to see into the rear, front or sides without moving either the top or the bottom.

Accessory models will continue to be added in the future.  Doing the major OEM accessories for the supported SBC is one long term goal.  The NopSCAD library is nice and I considered using it but it focuses on 3d printer design.  The fans are about the only thing relevant for SBC so I think just adding fan models to the existing library is the way to go for now.   Having accessory models for locating and checking can be important in some situations, which is why I suggested doing one of the GeeekPOE hat.
